packages/core/admin/admin/src/content-manager/components/InputJSON/index.js

```diff
@@ -79,6 +79,8 @@ class InputJSON extends React.Component {
     try {
       if (value === null) return this.codeMirror.setValue('');
 
+      if (value === 0) return this.codeMirror.setValue('0');
+
       return this.codeMirror.setValue(value);
```

Comment:
Would it maybe make more sense to stringify every number in line `84`? I'm wondering why `0` should be a special case ... 🤔

Reply:
I think there might be some boolean check in that function, so it sees 0 as `false`.
I have tried with other numbers and it works fine 🤷